server/api/controllers/article.js

```diff
@@ -42,9 +70,16 @@ const insertTag = async (tagArray) => {
 * @returns {Object} JSON object (JSend format)
 */
 export const createArticle = async (req, res) => {
-  try { 
-    const { body: { slug, description, body, references, categoryId } , user: { id: userId } } = req;
-    const newArticle = await Article.create({ slug, description, body, references, categoryId, authorId: userId });
+  try {
+    const {
+      body: {
+        description, body, references, categoryId
+      }, user: { id: userId }
+    } = req;
+    const slug = description.replace(/\s+/g, '-').toLowerCase();
```

Comment:
@jsamchineme Why is `slug` removed from the request payload ? Why is there a need to replace spaces and tabs with hyphens and the subsequent case conversion, is slug meant to be unique ?